packages/utils/number/handleNumericCallback/handleNumericCallback.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  removeThousandsDelimiter,
+  thousandsDelimiter,
+} from "../thousandsDelimiter/thousandsDelimiter";
+
+/**
+ * This function if the value is empty string or numerical, it calls the callback with the value as an argument.
```

Comment:
I think "This function executes the provided callback using the provided value if the value is an empty string or number." is a bit clearer.